packages/sdk-cli/src/models/HostConnections.ts

```diff
@@ -12,9 +15,37 @@ export class HostConnection {
     public host: RemoteHost,
   ) {}
 
+  static getDumpStreamTap() {
+    const dumpFileHandle = fs.openSync(process.env.FITBIT_DEVBRIDGE_DUMP_PATH as string, 'w');
```

Comment:
`dumpLogFile` instead of `dumpFileHandle`?

Reply:
File can be a bit confusing (is it a path? or is it a file descriptor/handle). Maybe `dumpLogFileHandle`?

Reply:
both work for me :)